compiler/condsyms.nim

```diff
@@ -77,7 +77,7 @@ proc initDefines*(symbols: StringTableRef) =
   defineSymbol("nimHasUserErrors")
   defineSymbol("nimUncheckedArrayTyp")
   defineSymbol("nimHasTypeof")
-  defineSymbol("nimErrorProcCanHaveBody")
+  defineSymbol("nimErrorProcCanHaveBody") # since #9665 but now unused
```

Comment:
```suggestion
  defineSymbol("nimErrorProcCanHaveBody")
```
Most of these are unused. This doesn't need a comment IMO

Reply:
that's not true, most of these are used. I've removed the comment anyhow, can be found with git blame